_drafts/A BLE Primer.md

```diff
@@ -0,0 +1,1202 @@
+
+Wireless provides the following benefits:
+
+- Removing wires.
+- Freedom and flexibility.
+- New use cases. Objects that cannot be connected with a wire, due to design factors or physical size limitations, can still be connected to other devices via wireless technologies.
+- Convenience, especially with remote or long-distance access.
+- It has the potential of saving lives! Think of a use case where an array of sensors are placed on the top of an electric pole, on the edge of a cliff, or even in a dam. Having the ability to collect data and control devices placed in these unsafe areas can reduce the safety risk significantly. 
+
+Of course nothing is perfect, so wireless has its drawbacks too:
+
+- Sacrificing quality in some cases (due to compression and limited bandwidth).
+- Reliability concerns.
+- Security (easier to sniff packets and listen in on communication when you don't need physical access to the device).
+- From a developer's perspective, it's much harder to develop and test wireless systems.
+- High power consumption (in some use cases).
+- Lack of interoperability in many cases (especially across different protocols and standards). This leads to more complicated integration between wireless networks.
+
+## Wireless Technologies and the Internet of Things (IoT)
+You've probably heard of the term IoT (Internet of Things) many times, and likely seen the hype in the media surrounding this somewhat confusing term. In basic terms, IoT refers to connecting devices to each other and to the Internet/Cloud (whether it's private or the mass Internet). It also implies wireless connectivity for devices and potentially adding connectivity to a slew of devices that were never connected before (even [flip-flops](https://www.geek.com/tech/smart-flip-flops-keep-brand-buyers-connected-1694504/)!).
+
+Wireless technologies are a dime a dozen, each with their own quirks and features. Some have been around for a long time, and some have only popped up recently. We'll spare the comparison between the different ones to a future post, but some of the most interesting and applicable wireless technologies to embedded developers are:
+
+- Wi-Fi
+- Bluetooth Classic & Bluetooth Low Energy
+- Zigbee
+- Z-Wave
+- LoRaWAN
+- ANT/ANT+
+- NFC
+- Infrared (IR)
+
+Most of these focus on the low-power aspect and are suitable for embedding into devices that need to run on small batteries for long periods of time. Applications include: industrial sensors, medical devices, wearables, and more.
+
+In today's post, we'll be covering one of the most popular low-power IoT technologies: Bluetooth Low Energy (BLE). We'll go over:
```

Comment:
You haven't yet designated an excerpt. I thing this line is as good a choice as any.

Reply:
Added.